example/.eslintrc

```diff
@@ -2,5 +2,8 @@
   "extends": "4catalyzer-react",
   "env": {
     "browser": true
+  },
+  globals: {
+    "css": false
```

Comment:
no globals!

Reply:
fine :p

Reply:
its using it in a "deprecated" way, I left the auto-assume global for the bare `css` to not break our code, but most every other lib requires an import, which i think makes sense